src/robotoff.js

```diff
@@ -126,5 +126,17 @@ export default {
     return axios.get(
       `${ROBOTOFF_API_URL}/images/logos?logo_ids=${logoIds.join(",")}`
     )
+  },
+
+  getNutritionValueFromImage(imageUrl) {
+    alert(' in robotoff.getNutritionValueFromImage')
+    var subStringUrl = imageUrl.substr(0, imageUrl.lastIndexOf("/"));
+    var url = `${ROBOTOFF_API_URL}/predict/nutrient?ocr_url=${subStringUrl}/10.json`
```

Comment:
```suggestion
    var url = `${ROBOTOFF_API_URL}/predict/nutrient?ocr_url=${subStringUrl}`
```
Shoudl already be in the url. The image id is not always 10